versions/1.0.7/build.gradle

```diff
@@ -19,15 +20,49 @@ dependencies {
     def prefix = "$kotlinBuildType:$kotlinBuild:kotlin-plugin-${pluginBuild}.zip!/Kotlin/kotlinc/lib"
     kotlinJVM tc("$kotlinBuildType:$kotlinBuild:internal/kotlin-compiler-before-shrink.jar")
     kotlinJVM tc("$kotlinBuildType:$kotlinBuild:kotlin-plugin-${pluginBuild}.zip!/Kotlin/lib/kotlin-plugin.jar")
-    kotlinJVM tc("$prefix/kotlin-runtime.jar")
-    kotlinJVM tc("$prefix/kotlin-reflect.jar")
-    kotlinJVM tc("$prefix/kotlin-test.jar")
-    kotlinJVM tc("$prefix/kotlin-jslib.jar")
     kotlinJS tc("$prefix/kotlin-jslib.jar")
+
+    kotlinLibs tc("$prefix/kotlin-runtime.jar")
+    kotlinLibs tc("$prefix/kotlin-reflect.jar")
+    kotlinLibs tc("$prefix/kotlin-test.jar")
+    kotlinLibs tc("$prefix/kotlin-jslib.jar")
+
+    /**
+     * ADD DEPENDENCIES TO KOTLIN 1.0.7 COMPILER
+     *
+     * If you want to add some another dependencies to kotlin-compiler you can use 'kotlinLibs' task.
+     *
+     * For example:
+     *
+     * kotlinUserLibs "your dependency from maven"
+     *
+     * FYI: Pay attention if yor library has got reflections, work with files and etc
+     * Please configure the 'executors.policy.template' in web-demo-backend.
+     * @see <a href="https://docs.oracle.com/javase/7/docs/technotes/guides/security/PolicyFiles.html">Java Security Police</a>
+     *
+     * HOW TO SET Java Security Police in 'executors.policy.template'
+     *
+     * If you want to add own dependency please use marker @WRAPPERS_LIB@
+     *
+     * For example:
+     *
+     * grant codeBase "file:@WRAPPERS_LIB@/junit-4.12.jar" {
+     *      permission java.lang.reflect.ReflectPermission "suppressAccessChecks";
+     *      permission java.lang.RuntimePermission "setIO";
+     *      permission java.io.FilePermission "<<ALL FILES>>", "read";
+     *      permission java.lang.RuntimePermission "accessDeclaredMembers";
+     * };
+     */
+
     compile fileTree(dir: projectDir.toString() + File.separator + "kotlin", include: '*.jar')
 }
 
 task copyKotlinLibs(type: Copy) {
     from configurations.kotlinJVM
     into projectDir.toString() + File.separator + "kotlin"
 }
+
+task copyLibs(type: Copy) {
```

Comment:
We don't need this task. We have to copy libraries only for Idea completion/highlighting/e.t.c. Idea don't need this libraries, so we can copy libs to war directly

Reply:
I created this task just for testing